scripts/deploy.sh

```diff
@@ -27,6 +27,12 @@ function run_ansible {
   openstack-ansible ${ANSIBLE_PARAMETERS} --forks ${FORKS} $@
 }
 
+apt-get install python-dev -y
+curl https://raw.githubusercontent.com/pypa/pip/7.1.2/contrib/get-pip.py -o /opt/get-pip.py
```

Comment:
I think we should pull the latest `get-pip.py` from `https://bootstrap.pypa.io/get-pip.py` and we should invoke it like so:

```
python get-pip.py install 'pip===7.1.2'
```

Assuming we want 7.1.2.